src/components/User/Assets/AssetRequests/components/AssetRequestList.vue

```diff
@@ -4,41 +4,41 @@
       <select-field
         class="arc-list__filter"
         v-model="filters.requestType"
-        label="Request type"
+        :label="'asset-request-list.lbl-request-type' | globalize"
       >
         <option
           v-for="requestType in Object.keys(ASSET_REQUEST_TYPES)"
-          :value="ASSET_REQUEST_TYPES[requestType].value"
+          :value="ASSET_REQUEST_TYPES[requestType]"
           :key="requestType"
         >
-          {{ ASSET_REQUEST_TYPES[requestType].text }}
+          {{ ASSET_REQUEST_TYPES[requestType] | assetRequestTypesFilter }}
```

Comment:
`requestType | assetRequestTypesFilter`
if necessary fix the filter

Reply:
Are you understand, why you using ASSET_REQUEST_TYPES const?

Reply:
The code what did you write will not work. Sort out why

Reply:
done